src/main/java/com/o19s/es/ltr/query/RankerQuery.java

```diff
@@ -206,6 +235,15 @@ public Weight createWeight(IndexSearcher searcher, ScoreMode scoreMode, float bo
             throw new IllegalStateException("LTR plugin is disabled. To enable, update ltr.plugin.enabled to true");
         }
 
+        try {
+            return _createWeight(searcher, scoreMode, boost);
+        } catch (Exception e) {
+            ltrStats.getStat(StatName.LTR_REQUEST_ERROR_COUNT.getName()).increment();
+            throw e;
+        }
+    }
+
+    private Weight _createWeight(IndexSearcher searcher, ScoreMode scoreMode, float boost) throws IOException {
```

Comment:
`_createWeight` is not a common Java naming convention , change to `createWeightInternal` ?